### PR TITLE
Optimism aura rebus

### PR DESCRIPTION
**Pull request by canhtrinh**

### CODE REVIEW

File: src/config/web3/cosmos/mainnet/aura.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    bech32PrefixValAddr: "aura" + "valoper",
+    bech32PrefixValPub: "aura" + "valoperpub",
+    bech32PrefixConsAddr: "aura" + "valcons",
+    bech32PrefixConsPub: "aura" + "valconspub",
+  },
+  currencies: [
+    {
+      coinDenom: "EAURA",
+      coinMinimalDenom: "ueaura",
+      coinDecimals: 6,
```
Comment (milapsheth) [inline src/config/web3/cosmos/mainnet/aura.ts:25]:
```suggestion
    {
      coinDenom: "AURA",
      coinMinimalDenom: "uaura",
      coinDecimals: 6,
```

File: src/config/web3/cosmos/mainnet/aura.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  ],
+  feeCurrencies: [
+    {
+      coinDenom: "EAURA",
+      coinMinimalDenom: "ueaura",
+      coinDecimals: 6,
+      gasPriceStep: {
+        low: 0.001,
+        average: 0.0025,
+        high: 0.004,
```
Comment (milapsheth) [inline src/config/web3/cosmos/mainnet/aura.ts:36]:
```suggestion
      coinDenom: "AURA",
      coinMinimalDenom: "uaura",
      coinDecimals: 6,
      gasPriceStep: {
        low: 0.001,
        average: 0.0025,
        high: 0.004,
```

File: src/config/web3/cosmos/mainnet/aura.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      },
+    },
+  ],
+  stakeCurrency: {
+    coinDenom: "EAURA",
+    coinMinimalDenom: "ueaura",
+    coinDecimals: 6,
```
Comment (milapsheth) [inline src/config/web3/cosmos/mainnet/aura.ts:43]:
```suggestion
    coinDenom: "EURA",
    coinMinimalDenom: "uaura",
    coinDecimals: 6,
```

File: src/config/web3/cosmos/mainnet/rebus.ts [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    coinImageUrl:
+      "https://app.rebuschain.com/public/assets/main/rebus-logo-single.svg",
+  },
+  features: [
+    "stargate",
+    "ibc-transfer",
+    "no-legacy-stdTx",
+    "ibc-go",
+    "eth-address-gen",
+    "eth-key-sign",
```
Comment (milapsheth) [inline src/config/web3/cosmos/mainnet/rebus.ts:71]: this is not needed, i saw the keplr config when connecting the wallet here (they had some eth like functionality for just some airdrop)
https://app.rebuschain.com/assets

File: src/config/web3/cosmos/mainnet/rebus.ts [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+export const rebus: CosmosChain = {
+  rest: "https://api.rebuschain.com:1317",
+  rpc: `${COSMOS_PROXY_RPC_MAINNET}/chain/rebus`,
+  chainId: "reb_1111-1",
+  chainIdentifier: "rebus",
+  chainName: "Rebus (EVM) v1.0.1",
+  bech32Config: Bech32Address.defaultBech32Config("rebus"),
+  bip44: {
+    coinType: 60,
```
Comment (milapsheth) [inline src/config/web3/cosmos/mainnet/rebus.ts:15]:
```suggestion
    coinType: 118,
```

File: src/config/web3/cosmos/mainnet/rebus.ts [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+import { COSMOS_PROXY_RPC_MAINNET } from "~/config/constants";
+
+import { CosmosChain } from "../interface";
+
+export const rebus: CosmosChain = {
+  rest: "https://api.rebuschain.com:1317",
+  rpc: `${COSMOS_PROXY_RPC_MAINNET}/chain/rebus`,
+  chainId: "reb_1111-1",
+  chainIdentifier: "rebus",
+  chainName: "Rebus (EVM) v1.0.1",
```
Comment (milapsheth) [inline src/config/web3/cosmos/mainnet/rebus.ts:12]:
```suggestion
  chainName: "Rebus",
```

File: src/config/web3/cosmos/mainnet/rebus.ts [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+    "ibc-transfer",
+    "no-legacy-stdTx",
+    "ibc-go",
+    "eth-address-gen",
+    "eth-key-sign",
+  ],
+  chainToAxelarChannelId: "channel-9",
+  explorer: "https://atomscan.com/rebus/",
+  walletUrlForStaking: "https://app.rebuschain.com/staking",
+  coinType: 60,
```
Comment (milapsheth) [inline src/config/web3/cosmos/mainnet/rebus.ts:76]:
```suggestion
  coinType: 118,
```

**Review by kalidux — APPROVED** [review]
LGTM

**Review by milapsheth — APPROVED** [review]

File: src/config/web3/cosmos/mainnet/aura.ts [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+      gasPriceStep: {
+        low: 0.001,
+        average: 0.0025,
+        high: 0.004,
+      },
+    },
+  ],
+  stakeCurrency: {
+    coinDenom: "EAURA",
+    coinMinimalDenom: "ueaura",
```
Comment (milapsheth) [inline src/config/web3/cosmos/mainnet/aura.ts:42]:
```suggestion
    coinMinimalDenom: "uaura",
```

File: src/config/web3/cosmos/mainnet/aura.ts [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+      coinDecimals: 6,
+      gasPriceStep: {
+        low: 0.001,
+        average: 0.0025,
+        high: 0.004,
+      },
+    },
+  ],
+  stakeCurrency: {
+    coinDenom: "EAURA",
```
Comment (milapsheth) [inline src/config/web3/cosmos/mainnet/aura.ts:41]:
```suggestion
    coinDenom: "AURA",
```

**Pull request merged**